api/types/src/convert.rs

```diff
@@ -264,11 +263,11 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
     pub fn try_table_item_into_write_set_change(
         &self,
         state_key_hash: String,
-        handle: TableHandle,
+        handle: u128,
```

Comment:
Similarly this is news to me, let's ensure that only changes related to the indexer are in this pr